### PR TITLE
[WIP] Feature/unit tests fixup

### PR DESCRIPTION
**Pull request by evanjtravis**
Reverts accidental changes from previous PR.

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -26,17 +26,14 @@ locals {
 # task_definition map
 locals {
   container_definition_file = "${lookup(var.task_definition, "container_definition_file", "containers.json")}"
-  cpu                       = "${lookup(var.task_definition, "cpu", var.launch_type == "FARGATE" ? "256" : "")}"
-  memory                    = "${lookup(var.task_definition, "memory", var.launch_type == "FARGATE" ? "512" : "0")}"
+  cpu                       = "${lookup(var.task_definition, "cpu", "256")}"
+  memory                    = "${lookup(var.task_definition, "memory", "512")}"
```
Comment (JonRoma) [inline locals.tf:30]: Shouldn't the `cpu` and `memory` values be numbers instead of strings?

File: locals.tf [hunk 2]
```diff
@@ -26,17 +26,14 @@ locals {
...
   task_role_arn             = "${lookup(var.task_definition, "task_role_arn", "")}"
 }
 
 # service_discovery map
 locals {
-  sd_container_name = "${lookup(var.service_discovery, "container_name", lookup(var.load_balancer, "container_name", ""))}"
-  sd_container_port = "${lookup(var.service_discovery, "container_port", lookup(var.load_balancer, "container_port", 0))}"
-
   dns_routing_policy = "${lookup(var.service_discovery, "routing_policy", "MULTIVALUE")}"
   dns_ttl            = "${lookup(var.service_discovery, "ttl", "60")}"
```
Comment (JonRoma) [inline locals.tf:38]: Should `ttl` be a number?

File: locals.tf (hunk 1 above)
Comment (ddriddle) [reply locals.tf:30]: This is old 0.11 code. There should be no changes whatsoever in the *.tf files. The point of the original pull request was to simplly add a test directory with tests for 0.11.  Unfortunately, mystery code changes were added with the test code. No idea where they came from. Neither does Evan. This pull request is to reversing the changes made, and make some minor changes to the test code.

**Review by ddriddle — APPROVED** [review]
This pull request diffs cleanly against v1.1.0. Thanks!

**Pull request merged**